css/styles.css

```diff
@@ -0,0 +1,146 @@
+body {
+  background-color: #f0f0f0;
+  font-family: "Open Sans", sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full-header {
+  flex-basis: 100%;
+}
+
+.main-container {
+  display: flex;
+  max-width: 800px;
+  align-items: center;
+  margin: auto;
+  justify-items: center;
+  padding-left: 1rem;
+}
+
+.head-container {
+  height: 3.5rem;
+  margin: auto;
+  align-items: center;
+  background-color: #562b80;
+}
+
+.containerimg {
+  height: 200px;
+  margin-bottom: 1rem;
+}
+
+img {
+  width: 95%;
+  height: 100%;
+  justify-content: center;
+  object-fit: cover;
+}
+
+h2 {
+  background-color: #562b80;
+  color: white;
+}
+
+h1 {
+  font-weight: 600;
+}
+
+label {
+  font-weight: 500;
+  font-size: 1.2rem;
+}
+
+header {
+  justify-content: center;
+  color: white;
+  padding-left: 1rem;
+  letter-spacing: 0.03rem;
+  font-size: 1.2rem;
+}
+
+input[type="text"] {
+  width: 100%;
+  height: 50%;
+}
+
+textarea {
+  width: 22rem;
+  height: 6rem;
+  border-color: #d4d4d4;
+  border: solid;
+  border-width: 0.1px;
+  border-radius: 0.3rem;
+  margin-top: 1rem;
+}
+
+input[type="text"],
+input[type="url"] {
+  width: 22rem;
+  height: 2rem;
+  border-color: #d4d4d4;
+  border: solid;
+  border-width: 0.1px;
+  border-radius: 0.3rem;
+}
+
+input {
+  margin-top: 0.5rem;
+  margin-bottom: 1rem;
+}
+
+button {
+  background-color: #562b80;
+  border-radius: 5px;
+  color: white;
+  height: 2rem;
+  width: 4rem;
+  border-style: none;
+  margin-top: 2rem;
+  margin-right: 1rem;
+  font-size: 1.05rem;
+}
+
+.justify-button {
+  justify-content: right;
+}
+
+@media screen and (min-width: 768px) {
+  input[type="text"], {
+    width: 100%;
+  }
+
+  input [type="url"] {
+    width: 100%;
+  }
+
+  textarea {
+    width: 100%;
+    height: 4rem;
+    border-color: #d4d4d4;
+    border: solid;
+    border-width: 0.1px;
+    border-radius: 0.3rem;
+    padding-left: 1rem;
+  }
+
+  img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .notes-text {
+    padding-left: 1rem;
+  }
+
+  .containerimg {
+    height: 100%;
+  }
+
+  header {
+    width: 850px;
```

Comment:
Why set a hard width on the `header`? You should just use the `container` class inside the `header` element to restrict the size fo the content